02-overview-surveys.Rmd

```diff
@@ -158,35 +192,44 @@ This is a simple example of how the presentation of the question and options can
 
 ### Disclosure {#overview-post-disclosure}
 
-Before data is released publicly, researchers need to ensure that individual respondents can not be identified by the data when confidentiality is required. There are a variety of different methods that can be used, including *data swapping*, *top or bottom coding*, *coarsening*, and *perturbation.* In data swapping, researchers may swap specific data values across different respondents so that it does not impact insights from the data but ensures that specific individuals cannot be identified. We can use *top and bottom coding* to mask extreme values. For example, researchers may top-code income values such that households with income greater than \$500,000 are coded into a single category of "\$500,000 or more". Other disclosure methods may include aggregating response categories or location information to avoid having only a few respondents in a given group and thus be identified. For example, researchers may use coarsening to display income in categories instead of as a continuous variable. We can also perturb the data by adding random noise. There is as much art as there is science to the methods used for disclosure. In the survey documentation, researchers should only provide high-level comments about the disclosure and not specific details. This ensures nobody can reverse the disclosure and thus identify individuals. For more information on different disclosure methods, please see @Skinner2009 and 
+Before data is released publicly, researchers need to ensure that individual respondents can not be identified by the data when confidentiality is required. There are a variety of different methods that can be used as described below:
+
+- **Data swapping**: Researchers may swap specific data values across different respondents so that it does not impact insights from the data but ensures that specific individuals cannot be identified
+- **Top/bottom coding**: A method used to mask extreme values. For example, researchers may top-code income values such that households with income greater than \$500,000 are coded as "\$500,000 or more" with other incomes being presented as integers between \$0 and \$499,999. This can impact analyses at the tails of the distribution.
+- **Coarsening**: Continuing with income as an example, a survey question may ask for a precise income but the public data may include data as a categorical variable. Geographic variables are often coarsened - the data collectors likely know the precise address of the respondent but the public data may only include the state or even region of the respondent.
+- **Perturbation**: Researchers may add random noise to outcomes. 
```

Comment:
```suggestion
- **Perturbation**: Researchers may add random noise to outcomes. As with swapping, this is done so that it does not impact insights from the data but ensures that specific individuals cannot be identified.
```